WebApiCoreSeed.UnitTests/Services/BaseServiceUnitTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Runtime.CompilerServices;
+using WebApiCoreSeed.Data.EF;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using WebApiCoreSeed.Data.Models;
+using System;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    public class BaseServiceUnitTest
+    {
+        public WebApiCoreSeedContext Context { get; set; }
+
+        public User User { get; set; }
+
+        [TestInitialize]

```

Comment:
I was talking with the guys ( @matipaulo  and @mdesagranja ) the other day about how we are using xUnit mostly on recent projects. 

I would like to gather feedback/thoughts about MSTest vs xUnit, and if you think we should enforce, recommend or leave it to each team preference.

I think we should strongly recommend using xUnit everywhere, unless there's a good reason not to (e.g. lots of legacy tests written with MSTest, or certain integration issues with other tools)

cc @AlphaGit @mravinale @matiasbeckerle @gastonmancini